server/src/config.rs

```diff
@@ -28,6 +38,75 @@ pub struct RuntimeConfig {
     pub background_thread_num: usize,
 }
 
+#[derive(Clone, Debug, Default, Deserialize)]
+#[serde(default)]
+pub struct StaticRouteConfig {
+    pub rule_list: RuleList,
+    pub topology: StaticTopologyConfig,
+}
+
+#[derive(Debug, Deserialize, Clone)]
+#[serde(default)]
+pub struct SchemaShardView {
+    pub schema: String,
+    pub auto_create_tables: bool,
+    pub default_engine_type: String,
+    pub default_timestamp_column_name: String,
+    pub shard_views: Vec<ShardView>,
+}
+
+impl Default for SchemaShardView {
+    fn default() -> Self {
+        Self {
+            schema: "".to_string(),
```

Comment:
would empty schema cause some issues? (I don't know, just confirm)

Reply:
It is just the default value for the `SchemaShardView` and the empty schema will be validated when used, so I think it is ok here.